packages/@ember/polyfills/tests/assign_test.js

```diff
@@ -49,7 +49,9 @@ moduleFor(
   'Ember.assign (polyfill)',
   class extends AssignTests {
     assign() {
-      return assignPolyfill(...arguments);
+      return expectDeprecation(() => {
```

Comment:
```suggestion
      expectDeprecation(() => {
```

If we can safely discard the return value of `assignPolyfill`'s result, then we simply don't need a return value.